lib/bundlex/cnode/server.ex

```diff
@@ -84,6 +84,12 @@ defmodule Bundlex.CNode.Server do
     {:stop, reason, state}
   end
 
+  def handle_info({:cnode, packet}, state) do
+    # forwards packet to caller
+    send state.caller, {:cnode, packet}
```

Comment:
Theoretically, a process (e.g. element) can manage multiple cnodes, so to make them distinguishable send https://hexdocs.pm/bundlex/Bundlex.CNode.html#t:t/0 instead of `:cnode`